document/CoreFormat.mdk

```diff
@@ -986,29 +1094,80 @@ A `location` object may contain a property named `tags`, whose value is a tag se
+A `physicalLocation` object represents the physical location where a result was detected.
+A physical location specifies a reference to a programming artifact together with a range of bytes or characters within that artifact.
 
-### `uri` property { #physicalLocationComponent-uri }
+### `uri` property { #physicalLocation-uri }
 
-A `physicalLocationComponent` object shall contain a property named `uri` whose value is a string that represents
-the location of the file to be modified as a valid URI.
+A `physicalLocation` object shall contain a property named `uri` whose value is a string that represents
+the location of the file as a valid URI.
 
-### `mimeType` property { #physicalLocationComponent-mimeType }
+The value of the `uri` property should be equal to the name of one of the properties on the
+`runInfo.fileInfo` object (&sect;[#runInfo-fileInfo]).
+`fileInfo` provides additional information about the file specified by `uri`.
 
-A `physicalLocationComponent` object should contain a property named `mimeType` with a string value that specifies
-the MIME content-type (RFC 2045) of the file.
+~ IsoExample
+
+```
+{
+  "version": "1.0",
+  "runLogs": [
+    {
+      "runInfo": {
+        "fileInfo": {
+          "file:///C:/Code/main.cc": [
+            {
+              "mimeType": "text/x-c",
+              "tags": [ "entryPoint" ]
+            }
+          ]
+        }
+      },
+      
+      "results": [
+        {
+          "ruleId": "CA2101",
+          "kind": "error",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "file:///C:/Code/main.cc",
+                "region: {
+                  "startLine": 24,
+                  "startColumn": 9
+                }
+              },
+              "logicalLocation": [
+                ...
+              ]
+            }
+          ]
+        }
+      ]
+    }
+  ]
+}
+```
+
+~
 
 ### `region` property { #physicalLocationComponent-region }
 
-A `physicalLocationComponent` object may have a property named `region` whose value is a `region` object (&sect;[#region]),
-that represents the region within a file where the result occurred.
-It should be present only on the last element of the physical location array in which the `physicalLocationComponent` occurs.
-This is because all the other elements of the array represent layers of packaging;
-only the last element represents the actual file within which the result occurred.
+A `physicalLocation` object may have a property named `region` whose value is a `region` object (&sect;[#region]),
+that represents the region within a file where the result was detected.
+
+If the result occurs in a nested file, then the `region` property shall specify the location of the result
+with respect to the innermost nested file.
+
+~ IsoExample
+If a result occurs in a C++ file contained in a Zip archive, then the region would represent
```

Comment:
I changed a few instances of "Zip" to "compressed". I left one instance, where the phrasing is "(as is the case for compressed containers such as Zip archives)". I don't actually know that every compressed archive format allows you to specify a contained file by way of a path.